### DISCUSSION TITLE
Confirming that SCENE_NUMBER is 1-indexed?

### DISCUSSION THREAD

**Discussion by vedantroy** [Breakthrough/PySceneDetect#405] [Q&A]
Just want to confirm that the format-string in something like this:
```
        scenedetect.split_video_ffmpeg(
            video_path,
            filtered_scene_list,
            output_dir=workdir,
            output_file_template="${SCENE_NUMBER}.mp4",
            show_progress=bool(os.environ.get("DEBUG", False)),
        )
```

is 1-indexed.

**Comment by Breakthrough** [comment]
Correct, the default implementation of `split_video_ffmpeg` uses 1-based scene indices.  You can specify a custom formatter if you want 0-based indices (see [the `formatter` arg](https://www.scenedetect.com/docs/0.6.4/api/video_splitter.html#scenedetect.video_splitter.split_video_ffmpeg) and [default impl](https://github.com/Breakthrough/PySceneDetect/blob/v0.6.4-release/scenedetect/video_splitter.py#L128)).
